index.html

```diff
@@ -3794,6 +3801,71 @@ <h3><dfn>Get Window Handles</dfn></h3>
 </aside>
 </section> <!-- /Get Window Handles -->
 
+<section>
+<h3><dfn>Create Window</dfn></h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/window/new</td>
+ </tr>
+</table>
+
+<p>Create a new <a>top level browsing context</a>.
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+ <li><p>If the implementation does not support creating new top-level
+  browsing contexts, return <a>error</a> with <a>error code</a>
+  <a>unsupported operation</a>.
+
+ <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
+  return <a>error</a> with <a>error code</a> <a>no such window</a>.
+
+ <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
+
+ <li><p>Let <var>type hint</var> be the result of <a>getting the property</a>
+  "<code>type</code>" from the <var>parameters</var> argument.
+
+ <li><p>Create a new <a>top-level browsing context</a> by running
+  the <a>window open steps</a> with <var>url</var> set to
+  "<code>about:blank</code>", <var>target</var> set to the empty string,
+  and <var>features</var> set to "<code>noopener</code>" and the user
+  agent configured to create a new browing context. If <var>type
+  hint</var> has the value "<code>tab</code>" and the implementation
+  supports multiple browsing context in the same OS window, the new
+  browsing context should share an OS window with the <a>current
+  browsing context</a>, otherwise the created browsing context should
+  be in a new OS window.
```

Comment:
It's not clear from the content if the tab should open as a foreground tab, or background tab. The same would apply to the window, if it should be opened as front-most, or background window. Given that the context hasn't been switched at this time, the current top-level browsing context would be in a background tab/window, if the new one will be the active one.

Also I think we need to apply s/browsing context/top-level browsing context/g to this paragraph.

Reply:
I think "new browsing context" is fine in context; we've already stated that it's a top-level browsing context and it's not necessary to keep repeating that fact.

I think whether the window/tab is focused is probably an implementation detail? *In theory* it doesn't matter, but I would *expect* that it isn't until you actually switch to it. Since we don't tell you to focus it or anything that matches the simplest reading of the current text.

Reply:
Note that it might affect the currently selected browsing context. Whether it stays in foreground or gets moved into the background could affect website operations. Like mentioned at TPAC eg for Facebook JS could be throttled.

As such it might be good to open the top-level browsing context as background tab, or as a window with a lower z-level as the current selected top-level browsing context's window.